media/redesign/stylus/components/wiki/revision-list.styl

```diff
@@ -97,3 +97,7 @@ html[dir=rtl] .revision-list-controls:before {
     width: 460px;
 }
 
+/* make the Compare selected revisions button align to layout */
+.revision-list-controls .fixed {
+    left: 24px;
```

Comment:
Needs to be RTL text friendly (example:https://developer.mozilla.org/he/docs/%D7%9E%D7%95%D7%A9%D7%92%D7%99%D7%9D$history)

We have a stylus mixin to help accomplish this:
https://github.com/mozilla/kuma/blob/d6692dff1d51c4529c4754874608041213250eca/media/redesign/stylus/includes/mixins.styl#L311-L327